src/qt/test/rpcnestedtests.cpp

```diff
@@ -35,14 +35,16 @@ static RPCHelpMan rpcNestedTest_rpc()
 }
 
 static const CRPCCommand vRPCCommands[] = {
-    {"test", &rpcNestedTest_rpc},
+    {"rpcNestedTest", &rpcNestedTest_rpc},
```

Comment:
fa2197c8b3178787d99e2acb5c3c717df14ddabf

nit, change ategory seems unnecessary.

Reply:
Ah right. :facepalm: I thought this is the name and needs to be adjusted, but yeah. Going to leave as is, because it doesn't matter.